synfig-studio/src/gui/canvasview.cpp

```diff
@@ -905,10 +904,13 @@ CanvasView::create_time_bar()
 	//time_window_scroll->set_can_focus(true); // Uncomment this produce bad render of the HScroll
 	time_window_scroll->show();
 
-	timetrack = manage(new class Gtk::VBox());
-	timetrack->pack_start(*widget_kf_list);
-	timetrack->pack_start(*timeslider);
-	timetrack->pack_start(*time_window_scroll);
+	timetrack = manage(new Gtk::Grid());
+	time_window_scroll->set_hexpand(true);
+	timetrack->attach(*time_window_scroll, 0, 0, 1, 1);
```

Comment:
We're not changing the value of the last two `attach(..., 1, 1)` parameters. Just remove those two parameters and let default argument values be used from the API instead.

That way code would be more readable :)

Reply:
Same for all other `grid->attach()` functions

Reply:
Actually @Keyikedalube , for now on Windows version, we need to set those last two parameters. Otherwise, it won't build :(

Reply:
Unfortunately I don't have Windows machine handy to test this.

Reply:
Understood :)

Reply:
Resolved...